lib/register.php

```diff
@@ -236,7 +236,7 @@ function gutenberg_collect_meta_box_data() {
 	 */
 	wp_add_inline_script(
 		'wp-editor',
-		'window._wpGutenbergEditor.initializeMetaBoxes( ' . wp_json_encode( $meta_box_data ) . ' )'
+		'window._wpGutenbergEditor.then( function( editor ) { editor.initializeMetaBoxes( ' . wp_json_encode( $meta_box_data ) . ' ) } );'
```

Comment:
Good catch here! Potential bug fixed. Maybe we should rename the variable to make it clear that it's a promise and not the editor instance. Something like `_wpLoadGutenbergEditor`

Reply:
👍 e156849c16f4f75a1f39647b91214b55059f5ff5